README.md

```diff
@@ -51,6 +49,7 @@ This will display:
     mjr:~/work/node_redis (master)$
 
 Note that the API is entirely asynchronous. To get data back from the server, you'll need to use a callback.
+From v.2.6 on the API supports camelCase and snack_case and all options / variables / events etc. can be used either way.
```

Comment:
Good hint. But all-caps is not supported for events, variables and options at the moment. Implementing this too would be easy, but I'm wondering if it might be worth thinking about removing support for it in v.3. It was implemented in the very first commit of node_redis but should it also be used that way? I'd disagree.
It is actually documented somewhere else but documenting it further might encourage people to use it, so removing it afterwards would be more work for users to switch.